### PR TITLE
Resolved comments: Removed unused constants

### PR DESCRIPTION
**Pull request by dobestler**
Upgrade to **ES 8.11.4** and querqy-for-lucene 5.7.lucene942.0 using Lucene 9.8.0

#109

### CODE REVIEW

File: src/main/java/querqy/elasticsearch/rewriter/SimpleCommonRulesRewriterFactory.java [hunk 1]
```diff
@@ -10,20 +10,31 @@
...
 import java.util.Map;
+import java.util.Optional;
 
 public class SimpleCommonRulesRewriterFactory extends ESRewriterFactory {
 
-    private static final SelectionStrategyFactory DEFAULT_SELECTION_STRATEGY_FACTORY =
-            new ExpressionCriteriaSelectionStrategyFactory();
+    public static final String CONF_IGNORE_CASE = "ignoreCase";
+    public static final String CONF_ALLOW_BOOLEAN_INPUT = "allowBooleanInput";
+//    public static final String CONF_BOOST_METHOD = "boostMethod";
```
Comment (renekrie) [inline src/main/java/querqy/elasticsearch/rewriter/SimpleCommonRulesRewriterFactory.java:26]: Thanks for adding these constants. Could you please remove the constants that are commented out?

File: src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java [hunk 2]
```diff
@@ -34,7 +34,6 @@ public void writeTo(final StreamOutput out) throws IOException {
         clearRewriterCacheResponse.writeTo(out);
     }
 
-    @Override
```
Comment (renekrie) [inline src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java:37]: This has a smell. If we can remove `Override`, this method is no longer being used. Can we remove it? Also, maybe the super class is now providing a new method that replaces `status()` and that we should override now?

File: src/main/java/querqy/elasticsearch/rewriterstore/PutRewriterResponse.java [hunk 3]
```diff
@@ -32,7 +33,6 @@ public void writeTo(final StreamOutput out) throws IOException {
         reloadResponse.writeTo(out);
     }
 
-    @Override
```
Comment (renekrie) [inline src/main/java/querqy/elasticsearch/rewriterstore/PutRewriterResponse.java:35]: See related comment in src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java

**Review by renekrie — CHANGES_REQUESTED** [review]
Thank you! Please see comments.

File: src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java (hunk 2 above)
Comment (dobestler) [reply src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java:37]: I thought so too. But it looks to me like the interface method was simply removed.

**Pull request merged**